perun-core/src/main/java/cz/metacentrum/perun/core/impl/AuthzResolverImpl.java

```diff
@@ -158,12 +160,19 @@ public AuthzRoles getRoles(User user) throws InternalErrorException {
 
 	public void initialize() throws InternalErrorException {
 
+		if(perun.isThisPerunReadOnly()) log.debug("Loading authzresolver manager init in readOnly version.");
```

Comment:
Same here, shouldn't we stop startup, when roles are missing and can't be created ?